delta/testkit/src/main/scala/ch/epfl/bluebrain/nexus/testkit/DeltaSpec.scala

```diff
@@ -0,0 +1,7 @@
+package ch.epfl.bluebrain.nexus.testkit
+
+import org.scalatest.{Inspectors, OptionValues}
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+
+abstract class DeltaSpec extends AnyWordSpecLike with Matchers with EitherValuable with OptionValues with Inspectors
```

Comment:
`BaseSpec` ?
We already have another one in integration tests and a `BaseRouteSpec` for routes

Reply:
I wanted to not call it BaseSpec just because we already have a BaseSpec and I was a little confusing. I don't mind what they are call, but I would like to make sure they have different names. Perhaps `BaseSpec` could become `BaseIntegrationSpec`?